spec/game_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+describe 'A game of bowling' do
```

Comment:
Nice, thoroughly tested

Reply:
Agreed, this are really good tests, focusing on behaviour.